vespa/deployment.py

```diff
@@ -973,20 +1295,31 @@ def get_endpoint(
         region: Optional[str] = None,
         environment: Optional[str] = "dev",
     ) -> str:
-        cluster_name = "{}_container".format(self.application_package.name)
+        # TODO: Support multiple endpoints.
+        auth_endpoints = []
         endpoints = self.get_all_endpoints(instance, region, environment)
         for endpoint in endpoints:
-            if endpoint["cluster"] == cluster_name:
-                authMethod = endpoint.get("authMethod", None)
-                if authMethod == auth_method:
-                    print(
-                        f"Found {auth_method} endpoint for {cluster_name}",
-                        file=self.output,
-                    )
-                    return endpoint["url"]
-        raise RuntimeError(
-            f"No {auth_method} endpoints found for container cluster {cluster_name}"
-        )
+            cluster = endpoint.get("cluster", None)
```

Comment:
Perhaps support cluster as an optional argument to the method?

Reply:
Agreed! ✅